geoalchemy2/types.py

```diff
@@ -161,6 +161,11 @@ class Geography(_GISType):
     """ The ``FromText`` geography constructor. Used by the parent class'
         ``bind_expression`` method. """
 
+    def column_expression(self, col):
+        # load column as WKB because ST_AsEWKB is not defined for geography
+        # see http://postgis.net/docs/ST_AsEWKB.html
+        return func.ST_AsBinary(col, type_=self)
```

Comment:
Instead of overriding the `column_expression` function it would be more consistent to introduce an `as_binary` property, as done with [`from_text`](https://github.com/geoalchemy/geoalchemy2/blob/master/geoalchemy2/types.py#L83-L85).